src/ResourceManager/Sql/Commands.Sql.Test/Commands.Sql.Test.Netcore.csproj

```diff
@@ -24,7 +24,6 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Microsoft.Azure.Management.Sql" Version="1.18.0-preview" />
```

Comment:
Instead of using the temp workaround, please just put the unpublished SDK in tools/LocalFeed.  This will allow you not to make all these changes to csproj, and when the package gets published all you will need to do is remove the localfeed package.

Reply:
Didn't know about this will do.